pymtl/tools/simulation/vcd.py

```diff
@@ -63,7 +63,7 @@ def mangle_name( name ):
 def write_vcd_signal_defs( o, model ):
 
   vcd_symbol = _gen_vcd_symbol()
-  all_nets   = set()
+  all_nets   = []
```

Comment:
I fixed this in https://github.com/cornell-brg/pymtl/pull/180, someone changed it back...